plugin/src/test/java/io/jenkins/plugins/coverage/metrics/steps/LineCoverageViewTest.java

```diff
@@ -0,0 +1,75 @@
+package io.jenkins.plugins.coverage.metrics.steps;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hamcrest.core.IsCollectionContaining;
+import org.junit.jupiter.api.Test;
+
+import io.jenkins.plugins.coverage.metrics.AbstractModifiedFilesCoverageTest;
+import io.jenkins.plugins.coverage.metrics.model.ChangedLinesModel;
+import io.jenkins.plugins.coverage.metrics.model.FileWithChangedLinesCoverageModel;
+import io.jenkins.plugins.coverage.metrics.model.Type;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.core.Is.*;
+
+/**
+ * TODO: JavaDoc me
+ */
+class LineCoverageViewTest extends AbstractModifiedFilesCoverageTest {
```

Comment:
The name of this test class does not really describe what is tested. After the business logic is seperated from the data classes, a test for specificely this business class can be written.
Example: MyBusinessClass.java -> MyBusinessClassTest.java